src/test/scala/dxWDL/dx/DxFileTest.scala

```diff
@@ -0,0 +1,73 @@
+package dxWDL.dx
```

Comment:
Thanks for adding the unit tests! Would it be possible to add a case where an `extraFields` parameter is passed, for example with "details" or/and "parts" and they are also returned in the describe result.